src/jgikbase/idmapping/service/mapper_service.py

```diff
@@ -126,6 +126,14 @@ def create_mapping(admin_ns, admin_id, std_ns, std_id):
                                         ObjectID(NamespaceID(std_ns), std_id))
         return ('', 204)
 
+    @app.route('/api/v1/mapping/<admin_ns>/<admin_id>/<std_ns>/<std_id>', methods=['DELETE'])
```

Comment:
I'm sure this is not worth changing at this point, but to me this is too many unlabeled parameters -- i'd rather have these four parameters in a JSON body with labels.

Reply:
I'm not sure what you mean by unlabeled - just that they're in the path and so aren't key:value like query params? Anyway, the update to bulk endpoints changes all this.